sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3745,6 +3745,21 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       }
     }
   }
+
+  test("SPARK-33593: Parquet vector reader incorrect with binary partition value") {
+    Seq(true).foreach(tag => {
+      withSQLConf("spark.sql.parquet.enableVectorizedReader" -> tag.toString) {
```

Comment:
`withSQLConf(SQLConf.PARQUET_VECTORIZED_READER_ENABLED.key -> "true") {`?

Reply:
> `withSQLConf(SQLConf.PARQUET_VECTORIZED_READER_ENABLED.key -> "true") {`?

Miss one point, for debug I remove `false`